### ISSUE TITLE
Add on backup size

### ISSUE THREAD

**Issue by jane-t** [Draggon/hassio-hdd-tools#34]
I noticed my daily backup has exploded in size since this was installed.

Would you expect the tar.gz for hdd_tools to be 100Mb?

**Comment by McGiverGim** [comment]
You are right. Something is wrong with the addon to create this big file. I'm not too sure what is the origin of the problem right now.

**Comment by jane-t** [comment]
Out of interest, I looked and the hdd-tools is by far the biggest thing in my daily backup.   As a test I uninstalled it and reinstalled it, to see if a clean install was smaller and it was not.
 
![2021-08-02_153831](https://user-images.githubusercontent.com/920792/127878974-27c57017-d0e6-442b-be33-431bfa42353a.jpg)

**Comment by McGiverGim** [comment]
I've investigated a little more on this. It seems the backup is storing the "image" of the docker and for other addons it does not.

I'm not an expert in docker but what it seems is that other addons upload the image to some repo, but hdd tools does not, so the backup tool stores the image but for others it does not because it can download it from the online repo:
![image](https://user-images.githubusercontent.com/2673520/128001619-4bf8c9d0-8c1a-4187-b3fb-7493b1647774.png)

I don't know if someone with more knowledge in docker and home assistants addons knows how to fix this.

**Comment by chertvl** [comment]
Hey, any news?

**Comment by McGiverGim** [comment]
Not for my part :(
